instruments/aiia.yaml

```
@@ -0,0 +1,1241 @@
+    links: [ { "urn": "urn:nl:aivt:ir:iama:1.0:2B.1.4"} ]
+    required_status: No
+    template: *assessmentcard_template
+  - question: "Is het AI-systeem betrouwbaar?"
+    urn: "urn:nl:aivt:ir:aiia:1.0:3.2"
+    labels:
+      - key: topic
+        value: "Betrouwbaarheid"
+    type: [ "open_question" ]
+    suggested_roles: ["Data scientist", "Domeinspecialist", "Opdrachtgever", "Projectleider"]
+    lifecycle: ["Verificatie en validatie"]
+    links: [ {"urn": "urn:nl:aivt:ir:iama:1.0:2B.3.6"} ]
+    required_status: Yes
+    template: *assessmentcard_template
+  - question: "Wat zijn de belangrijkste factoren die de prestaties van het AI-systeem beïnvloeden?"
+    urn: "urn:nl:aivt:ir:aiia:1.0:3.2.1"
+    labels:
+      - key: topic
+        value: "Accuraatheid"
+    type: [ "open_question" ]
+    suggested_roles: ["Data scientist", "Domeinspecialist", "Opdrachtgever", "Projectleider"]
+    lifecycle: ["Ontwikkelen", "Verificatie en validatie"]
+    links: [ { } ]
+    required_status: No
+    template: *assessmentcard_template
+  - question: "Wordt een deel van de (sub)dataset uitgesloten voor het leren van het model en alleen gebruikt voor het bepalen van de betrouwbaarheid of wordt de betrouwbaarheid van het model berekend met behulp van cross-validatie?"
+    urn: "urn:nl:aivt:ir:aiia:1.0:3.2.2"
+    labels:
+      - key: topic
+        value: "Accuraatheid"
+    type: [ "open_question" ]
+    suggested_roles: ["Data scientist", "Domeinspecialist", "Opdrachtgever", "Projectleider"]
+    lifecycle: ["Dataverkenning en datapreparatie", "Ontwikkelen"]
+    links: [ { } ]
+    required_status: No
+    template: *assessmentcard_template
+  - question: "Hoe is de (hyper)parameter-tuning onderbouwd en getoetst?"
+    urn: "urn:nl:aivt:ir:aiia:1.0:3.2.3"
+    labels:
+      - key: topic
+        value: "Accuraatheid"
+    type: [ "open_question" ]
+    suggested_roles: ["Data scientist", "Domeinspecialist", "Opdrachtgever", "Projectleider"]
+    lifecycle: ["Ontwikkelen"]
+    links: [ { } ]
+    required_status: No
+    template: *assessmentcard_template
+  - question: "Hoe is het AI-systeem technisch geïmplementeerd?"
+    urn: "urn:nl:aivt:ir:aiia:1.0:3.3"
+    labels:
+      - key: topic
+        value: "Technische implementatie"
+    type: [ "open_question" ]
+    suggested_roles: ["Data scientist", "Domeinspecialist", "Opdrachtgever", "Projectleider"]
+    lifecycle: ["Ontwerp", "Ontwikkelen"]
+    links: [ {"urn": "urn:nl:aivt:ir:iama:1.0:2A.1.1" }, {"urn": "urn:nl:aivt:ir:iama:1.0:2B.1.1"}]
+    required_status: Yes
+    template: *assessmentcard_template
+  - question: " Is er nagedacht hoe het AI-systeem past in de al bestaande technische- en systeeminfrastructuur en zijn hier passende maatregelen voor genomen om deze uit te rollen (indien van toepassing)?"
+    urn: "urn:nl:aivt:ir:aiia:1.0:3.3.1"
+    labels:
+      - key: topic
+        value: "Technische implementatie"
+    type: [ "open_question" ]
```

Perhaps we could also allow to upload a document for this question?
```suggestion
    type: [ "open_question", "upload_document" ]
```